tests/acceptance/features/apiActivities/activities.feature

```diff
@@ -3208,3 +3208,544 @@ Feature: check activities
     And user "Alice" has uploaded file with content "updated ownCloud test text file" to "textfile.txt"
     When user "Brian" tries to list the activities of file "textfile.txt" from space "Personal" owned by user "Alice" using the Graph API
     Then the HTTP status code should be "403"
+
+
+  Scenario: check activities of destination file and folder after copying a file into it
```

Comment:
can't we use resource word rather than file and folder

Reply:
Yeah, we can. I was just trying to be specific with file and folder.